iis/tox.ini

```diff
@@ -10,6 +10,8 @@ envdir =
     py27: {toxworkdir}/py27
     py38: {toxworkdir}/py38
 dd_check_style = true
+description =
+    py{27,38}: e2e ready
```

Comment:
Since we're just running e2e on py38 you can remove `py27`. Or you can try asserting for the old IIS service check name when the environment is py27.

Reply:
If you can assert the service check in py2 as well that would be great! (example of distinguishing what tests are run on what python version: [hyperv](https://github.com/DataDog/integrations-core/blob/master/hyperv/tests/test_hyperv.py))

Reply:
I'll see if I can get that working. Thanks for the example Sarah!